src/main/scala/org/scalabridge/sitegen/StaticSiteGenerator.scala

```diff
@@ -28,6 +28,21 @@ object StaticSiteGenerator {
     }
   } yield Underlined(value)
 
+  val strongParser: Parsley[AST] = between("**") map rightToNode("strong")
+
+  def rightToNode(tagName: String)(chars: Seq[Char]): AST = {
+    val str = chars.mkString
+    NonEmptyString.from(str) match {
+      case Right(v) => mkNode(v, tagName)
+      case _        => throw new Exception(s"Empty string in $tagName")
+    }
+  }
```

Comment:
We should make sure to return well formed errors, in the shape of `Either`s, so that we can make the application fail gracefully when encountering errors